main.go

```diff
@@ -49,7 +47,7 @@ type Log struct {
 // Vars and Constants
 // ============================
 
-var logFilePath = path.Join(os.Getenv("HOME"), "/Dropbox/The Heap/Beacon/beacon_log.json")
+// var logFilePath = path.Join(os.Getenv("HOME"), "/Dropbox/The Heap/Beacon/beacon_log.json")
```

Comment:
Ah good catch, thanks for reviewing @jbydeley ! Will fix now! (and add you as a contributor for doing a PR :)